exercises/00-init/src/modules/users/users-saga.js

```diff
@@ -0,0 +1,35 @@
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
+
+import {getUsers as getUsersEffect, addUser as addUserEffect} from 'modules/users/users-effects'

```

Comment:
Good solution, but might get out of hand for more imports. If this got bigger, I would consider either naming the functions themselves with the `Effect` suffix; or possibly do this:
```
import * as UserEffects from 'modules/users/users-effects'
```
Then in the code you would use it as e.g. `UserEffects.addUser(...)`